libcloudforensics/providers/gcp/internal/common.py

```diff
@@ -207,7 +207,7 @@ def BlockOperation(self,
 
 def ExecuteRequest(client: 'googleapiclient.discovery.Resource',
                    func: str,
-                   kwargs: Dict[str, str],
+                   kwargs: Union[Dict[str, str], Dict[str, Dict[str, str]]],
```

Comment:
To be consistent with the rest of the codebase I would rather type hint such `Dict` as `Dict[str, Any]` (and thus get rid of the `Union` and corresponding import) otherwise it quickly gets lengthy, wdyt?

Reply:
sgtm, done.